src/Adapters/Wp/Composites/CompositeAdapter.php

```diff
@@ -364,10 +364,11 @@ public function getTranslations(): ?Collection
             } else {
                 $composite = WpModelRepository::instance()->getPost($compositeId);
             }
-            if ($composite instanceof \WP_Post) {
+            $compositeUrl = parse_url(get_permalink($composite));
+            if ($composite instanceof \WP_Post && $compositeUrl['path'] !== '/') {
```

Comment:
Why not just check if it's a WP_Post and it's published?
```suggestion
            if ($composite instanceof \WP_Post && $composite->post_status === 'publish') {
```